golem/resources/proposal/proposal.py

```diff
@@ -1,14 +1,15 @@
 import asyncio
-from dataclasses import dataclass
 from datetime import datetime, timedelta, timezone
-from typing import TYPE_CHECKING, AsyncIterator, Literal, Optional, Union
+from typing import TYPE_CHECKING, AsyncIterator, Optional, Union, cast
 
 from ya_market import RequestorApi
 from ya_market import models as models
 
-from golem.payload import Constraints, Properties
+from golem.payload import Constraints, PayloadSyntaxParser, Properties
+from golem.payload.defaults import NodeInfo
```

Comment:
`NodeInfo` should be importable from `golem.payload` like this
`from golem.payload import NodeInfo`, please update the `golem.payload.__init__.py` if needed

Reply:
well, I can update the `__init__` but I didn't touch `__init__` in this pull request...

Reply:
We need it in the __init__ so user dosen't have to know from where to import NodeInfo so if it is not a problem pleas add it, in this module use import you think is best in your opinion

Reply:
gotcha, the only issue I had is that this pull request didn't touch `golem.payload`...